django_tables2/columns/templatecolumn.py

```diff
@@ -18,6 +18,7 @@ class TemplateColumn(Column):
     Arguments:
         template_code (str): template code to render
         template_name (str): name of the template to render
+        extra_context (dict): template variables used to (optional)
```

Comment:
This sentence seems to be incomplete.

Reply:
@jieter , Can you suggest a full content of line? I do not know enough English to do it myself.

Reply:
What about:
```
extra_content (dict): optional extra template context.
```